Makefile

```diff
@@ -200,7 +200,7 @@ test/reference: test/common
 # checkout Cockpit's PF/React/build library; again this has no API stability guarantee, so check out a stable tag
```

Comment:
The "Update cockpit's lib to 260" commit mentions "podman"; perhaps say "our CSS"?

Reply:
Copy pasta-error, adjusted.